cypress/e2e/webTables.cy.js

```diff
@@ -1,7 +1,120 @@
 /// <reference types='cypress' />
+const { faker } = require('@faker-js/faker');
+
+const username = faker.internet.userName();
+const lastname = faker.name.lastName();
+const email = faker.internet.email();
 
 describe('Web Tables page', () => {
-  it('', () => {
+  beforeEach(() => {
+    cy.visit('https://demoqa.com/webtables');
+  });
+  it('should check Pagination', () => {
```

Comment:
Please consider that describe + test name should create summary together - in this case we have 
_Web Tables page + should check Pagination // Web Tables shouldn't check pagination._ 

Better to write 
_Web Tables page + should allow user to change page_ 

Check and fix everywhere

Reply:
+